css/src/base.less

```diff
@@ -453,6 +453,22 @@ table.floatThead-table {
     margin: 10px;
 }
 
+.task-status-ok {
+  color: #9da827;
+}
+
+.task-status-submitted {
+  color: #3498db;
+}
+
+.task-status-wrong {
+  color: #f39c12;
+}
+
+.task-status-empty {
+  color: #aaaaaa;
+}
+
```

Comment:
please use the variables we defined for these colors in `variables.less`. And use one of our defined grays that we already use for the `.task-status-empty`